sdcflows/workflows/fmap.py

```diff
@@ -52,70 +50,46 @@ def init_fmap_wf(omp_nthreads, fmap_bspline, name='fmap_wf'):
     outputnode = pe.Node(niu.IdentityInterface(fields=['fmap', 'fmap_ref', 'fmap_mask']),
                          name='outputnode')
 
-    # Merge input magnitude images
-    magmrg = pe.Node(IntraModalMerge(), name='magmrg')
+    prepare_magnitude_wf = init_prepare_magnitude_wf(omp_nthreads=omp_nthreads)
```

Comment:
Okay, now I got it. Still not sure b0 is a good name for the other file. Maybe gre?